.travis.yml

```diff
@@ -1,15 +1,22 @@
 language: scala
-script:
-  - sbt ++$TRAVIS_SCALA_VERSION sourcecodeJVM/test:run sourcecodeJS/test:run
+
 scala:
-  - 2.10.6
-  - 2.11.8
-  - 2.12.0
+  - 2.11.11
+
+sudo: true
+
 jdk:
-  - openjdk7
   - oraclejdk8
-matrix:
-  exclude:
-    - scala: 2.12.0
-      jdk: openjdk7
-sudo: false
+
+before_install:
```

Comment:
you need to install re2 see https://github.com/scala-native/scala-native/blob/master/.travis.yml#L28-L38